src/game/puppyprint.c

```diff
@@ -878,148 +1182,269 @@ void print_small_text(s32 x, s32 y, const char *str, s32 align, s32 amount, s32
             }
         }
 
-        gSPScisTextureRectangle(gDisplayListHead++, ((x + shakePos[0] + textPos[0]     ) << 2),
-                                                    ((y + shakePos[1] + offsetY + textPos[1] + wavePos) << 2),
-                                                    ((x +  textPos[0] + shakePos[0] + 8) << 2),
-                                                    ((y + wavePos + offsetY + shakePos[1] + 12 + textPos[1]) << 2),
-                                                    G_TX_RENDERTILE, (textX << 6), (textY << 6), (1 << 10), (1 << 10));
+        gSPScisTextureRectangle(gDisplayListHead++, (x + textPos[0]) << 2,
+                                                    (y + textPos[1] + offsetY) << 2,
+                                                    (x + textPos[0] + 8) << 2,
+                                                    (y + textPos[1] + offsetY + 12) << 2,
+                                                    G_TX_RENDERTILE, (textX << 6), 0, 1024, 1024);
```

Comment:
For _some reason_, GlideN64 does not play well when `textX` is set to exactly 256 (in-game, lowercase `j`'s end up getting replaced with `!`'s apparently). This seems to be a non-issue if you add 1 to the value after the bitshift, and doing this doesn't seem to affect console or other plugins I've tried, but it's still bizarre.